configs/dardel.config

```diff
@@ -38,6 +38,11 @@ def clusterOptionsCreator = { mem, time, cpus ->
 includeConfig 'tool_resources.config'
 
 process {
+    resourceLimits = [
```

Comment:
Can you add a comment here so we know we're limiting to a shared partition, and that this should be overridden for a process needing to use another partition.